services/QuillLMS/client/app/bundles/Grammar/components/grammarActivities/question.tsx

```diff
@@ -190,7 +192,8 @@ export class QuestionComponent extends React.Component<QuestionProps, QuestionSt
     if (Object.keys(responses).length) {
       if (response !== '') {
         const trimmedResponse = response.replace(/<\/?[^>]+(>|$)/g, '').replace(/&nbsp;/g, ' ')
-        if (!isFirstAttempt && response === question.attempts[0].text) {
+        const firstAttemptResponse = !isFirstAttempt ? extractResponse(question.attempts[0]).text : null;
```

Comment:
Take or leave: consider removing the negated conditional:

`const firstAttemptResponse = isFirstAttempt ? null : extractResponse(question.attempts[0]).text;`